### ISSUE TITLE
Enrollment Accessibility: Shopping Cart class sched section

### ISSUE THREAD

**Issue by leslieg** [ucsc/AIS-Mobile#36]
Shopping Cart - scroll to bottom "My [term] Class Schedule"

Example: W1306243
ANTH 135A-01 (class)
ANTH 135A-01A (section)
1.  **The real classes (not sections): each element in the entire section is considered a link. Can we wrap a link around the text below? E.g. each class listing says "link" like 12 times. I think the title should be a link so they don't have to listen to the whole thing.  or?
2.  **There is nothing other than a visual cue that the status is enrolled, dropped or waitlisted. On the desktop version, there is a column in the table with header called "status" with a symbol in it. It would be nice to have the image have an alt tag that said "enrolled" or whatever appropriate status.
3. The "^" on the collapsible sections identifies again as the heading level and a link - so not only the title of the collapsible section but also the image.  Wrap all in link?
4. (not sure of priority here) Also in these real classes, the labels (like Description, Days/Times, Room, etc.) don't appear to be identified as such.  The value next to it always starts with "colon".  With sections on the other hand, which aren't links, the entire thing is read as one and each label is read as a label: nowhere does it say "colon".  
5. (JUST A NOTE)  Expandable and collapsible sections don't identify as such....on voice over but they do when the screen reader is going so I think it's coded correctly...

**Comment by leslieg** [comment]
The class search (three items) are not labeled as links or buttons.

**Comment by leslieg** [comment]
The "You are here:' at the top, it would be nice if the "1" said "page 1 of 3 - link" etc.rather than "you are here - link"   The others say "2" and "3" - would be much clearer if 2 and 3 weren't spoken or were spoken and it said "greyed" out or whatever is typical.

**Comment by leslieg** [comment]
Problem when coming back from class search using the back button - it goes to a blank page (which hopefully we can fix but if not, there is no indication what just happened.)

**Comment by leslieg** [comment]
Last collapsible on page is `<H2>` but the others are `<H3>`  should they all be `<h2>`?

**Comment by leslieg** [comment]
Is there a way to identify visual legends as such?  When scrolling down the page, you either come across

**Comment by leslieg** [comment]
Your shopping cart is empty, heading level 3?
